autopts/pybtp/defs.py

```diff
@@ -24,6 +24,10 @@
 BTP_SERVICE_ID_MESH = 4
 BTP_SERVICE_ID_MMDL = 5
 BTP_SERVICE_ID_GATTC = 6
+BTP_SERVICE_ID_VCS = 7
```

Comment:
this is conflicting with gatt server service

Reply:
done